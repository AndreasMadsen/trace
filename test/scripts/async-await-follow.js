async function f1() {
  await Promise.resolve();
  throw new Error('FAIL');
}

async function f2() {
  await Promise.resolve()
  await f1();
}

async function f3() {
  await Promise.resolve();
  try {
    await f2();
  } catch (err) {
    console.log(err);
  }
}

f3();
