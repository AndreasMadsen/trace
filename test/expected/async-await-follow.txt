Error: FAIL
    at f1 (/scripts/async-await-follow.js:3:9)
    at ____________________:0:0
    at ____________________:0:0
    at f1.<anonymous> (<anonymous>)
    at f1 (/scripts/async-await-follow.js:2:17)
    at ____________________:0:0
    at <anonymous>
    at f1 (/scripts/async-await-follow.js:1:80)
    at f2 (/scripts/async-await-follow.js:8:9)
    at ____________________:0:0
    at ____________________:0:0
    at ____________________:0:0
    at ____________________:0:0
    at f2.<anonymous> (<anonymous>)
    at f2 (/scripts/async-await-follow.js:7:17)
    at ____________________:0:0
    at <anonymous>
    at f2 (/scripts/async-await-follow.js:6:18)
    at f3 (/scripts/async-await-follow.js:14:11)
    at <anonymous>
    at ____________________:0:0
    at ____________________:0:0
    at f3.<anonymous> (<anonymous>)
    at f3 (/scripts/async-await-follow.js:12:17)
    at ____________________:0:0
    at <anonymous>
    at f3 (/scripts/async-await-follow.js:11:18)
    at Object.<anonymous> (/scripts/async-await-follow.js:20:1)
